pkg/querier/queryrange/roundtrip.go

```diff
@@ -232,7 +232,7 @@ func getOperation(path string) string {
 		return QueryRangeOp
 	case strings.HasSuffix(path, "/series"):
 		return SeriesOp
-	case strings.HasSuffix(path, "/labels") || strings.HasSuffix(path, "/label"):
+	case strings.HasSuffix(path, "/labels") || strings.HasSuffix(path, "/label") || strings.HasSuffix(path, "/values"):
```

Comment:
Q: Does anyone know why middeware chain was disabled for `/labels/<name>/values` endpoint on the query frontend?

We in fact have a test to lock that behaviour.
https://github.com/grafana/loki/blob/main/pkg/querier/queryrange/roundtrip_test.go#L390-L400

I wanted this support so that middlewares like `metrics.go` stats collector can work on this endpoints too. (#5971)

Any thoughts?

Reply:
If no protest, I can remove that test!